modules/sampler_hijack.py

```diff
@@ -234,21 +234,28 @@ def __call__(self, input_ids: torch.LongTensor, scores: torch.FloatTensor) -> to
             input_ids = input_ids[:, -self._range:]
 
         for input_ids_row, scores_row in zip(input_ids, scores):
+
+            # Use normal Python data types for improved performance
+            input_ids = input_ids_row.tolist()
+            
             # Raw integer must be extracted here to check for set membership.
```

Comment:
This comment doesn't make sense anymore since the list elements are already raw integers.